tasks/Module 3/task_1.py

```diff
@@ -0,0 +1,36 @@
+"""Sandwich calculator."""
+import pyinputplus as pyip
+
+BREAD_COST = {"wheat": 1, "white": 1.5, "sourdough": 1.5}
+MEAT_COST = {"chicken": 3, "turkey": 4, "ham": 3, "tofu": 4}
+CHEESE_COST = {"cheddar": 2, "swiss": 1.5, "mozzarella": 1}
+SAUSE_COST = {"mayo": 0.2, "mustard": 0.1, "lettuce": 0.2, "tomato": 0.1}
+
+
+def sandwich_maker():
```

Comment:
Usually we use the convention of `verb_item` eg `make_sandwiches` instead of `sandwiches_maker`. This way it's clear what's the main purpose of our functions/code.